securitas/form/login_user.py

```diff
@@ -0,0 +1,13 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField
+from wtforms.validators import DataRequired
+
+
+class LoginUserForm(FlaskForm):
```

Comment:
Is it worth trying to encode some rules here about what constitutes a valid username/password? We'll basically be re-encoding the IPA rules I guess, but the advantage is that some of them could be verified on the client side before the form is ever submitted (e.g. min/max length).

Reply:
Yeah I asked myself that question (for the registration form), and I thought that if we want to change the password policy we'd have to do it in two different places, which is error-prone.
So I've only left basic "not-empty" validators.